sw/device/lib/testing/test_framework/ottf_isrs.S

```diff
@@ -170,9 +170,10 @@ handler_irq_software:
   jal compute_mepc_on_synchronous_irq
   sw t0, 0(sp)
 
-  // Store stack pointer to current TCB (only if concurrency is enabled, i.e.,
-  // the test that triggers this is running as a FreeRTOS task).
-  jal save_current_sp_to_tcb
+  // Save MEPC to the stack.
+  // NOTE: this IRQ is asynchronous, therefore, we do not need to modify MEPC.
+  csrr t0, mepc
+  sw t0, 0(sp)
```

Comment:
Did you mean to replace the block above?

Reply:
Oh.... right..............................